index.html

```diff
@@ -163,7 +176,9 @@ <h1>About  <br>Myself</h1>
                             <li class="list-item">
                                 <img src="assets/css.png" alt="">
                                 HTML</li>
-                            <li class="list-item"><img src="assets/html.png" alt="">CSS</li>
+                            <li class="list-item">
+                              <img src="assets/html.png" alt="">
+                              CSS</li>
```

Comment:
- [ ] Kindly fix the images displayed for HTML and CSS. You can see below, that you have mistakenly added HTML image to CSS and CSS to HTML. Kindly fix it.

https://user-images.githubusercontent.com/70269507/171372916-dab4d523-c63c-4be0-b886-d52e0b573e75.mov